scripts/citests.sh

```diff
@@ -1,30 +1,28 @@
 #!/bin/bash
 
-set -e
+set -eux
 
-#
-# Splits test stage tasks between different CircleCI Nodes to improve performance.
-#
+mkdir -p $CIRCLE_ARTIFACTS $CIRCLE_TEST_REPORTS
 
-echo "Running node $CIRCLE_NODE_INDEX"
+./scripts/deploystorybook.sh
+git checkout $CIRCLE_SHA1
 
-if [ "$CIRCLE_NODE_INDEX" -eq "0" ]; then
```

Comment:
is parallelization handled differently?

Reply:
yup, it's gone. I didn't want to mess with it. this was simpler and similar time wise given picturebook 2.0 is faster